<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logging</class>
 <widget class="QMainWindow" name="Logging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>查询条件</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="6">
        <widget class="QDateTimeEdit" name="dateTimeEdit_end">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QDateTimeEdit" name="dateTimeEdit_begin">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>级别</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_level">
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fatal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>从</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="pushButton_search">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
